docker/docker-compose.dev.yml

```diff
@@ -12,7 +12,22 @@ services:
     volumes:
       - ..:/srv/app
     command: "flask run --host=0.0.0.0"
+    networks:
+      - esnet
     ports:
       - "5000:5000"
     env_file:
       - dev.env
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.6.1
+    networks:
+      - esnet
+    ports:
+      - 9200:9200
+      - 9300:9300
```

Comment:
Need to dig in the documentation, but IIRC port 9200 exposes the API (in fact the Python code talks to that) where as 9300 is for multiple Elasticsearch instances to communicate (replication, sharding, etc). The latter port then may not be needed here because we only have a single container.

Reply:
I've removed this port from `ports` as it's not being used.